packages/design-system/src/components/button.tsx

```diff
@@ -51,31 +51,31 @@ const perColorStyle = (variant: ButtonColor) => ({
   background: variant === "ghost" ? "transparent" : backgrounds[variant],
   color: foregrounds[variant],
 
-  "&[data-button-state=auto]:hover, &[data-button-state=hover]": {
+  "&[data-state=auto]:hover, &[data-state=hover]": {
```

Comment:
Now that we handle `data-state` prop explicitly rather than just passing it through it safe to use it here